roles/lib_openshift/library/oc_route.py

```diff
@@ -1707,14 +1704,14 @@ def run_ansible(params, check_mode=False):
         if params['tls_termination'] and params['tls_termination'].lower() != 'passthrough':  # E501
 
             for key, option in files.items():
-                if key == 'destcacert' and params['tls_termination'] != 'reencrypt':
+                if not option['path'] and not option['content']:
```

Comment:
@joelddiaz it is in a different PR (#4887). Just cherry-picked it so this PR is complete for itself.
Thanks for the generation note. I fixed the commit in the other PR and rebased.